src/Components/AlertBox/AlertBox.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './AlertBox.css';
+import Typography from '@material-ui/core/Typography';
+import Card from '@material-ui/core/Card';
+import { CardContent } from '@material-ui/core';
+
+export class AlertBox extends React.Component {
```

Comment:
Could probably turn this into a function instead of a class (unless we decide to add some more logic/methods to this)